Applications/CLI/Tests.cmake

```diff
@@ -134,6 +134,30 @@ if(NOT OGS_USE_MPI)
             DIFF_DATA
             line_1_line_${mesh_size}.vtu line_${mesh_size}_neumann_pcs_0_ts_1_t_1.000000.vtu D1_left_N1_right pressure
         )
+
+        AddTest(
+            NAME GroundWaterFlowProcess_line_1_Robin_Right_Picard_${mesh_size}
```

Comment:
Please add a small description of the benchmark to ["Selected benchmarks"](https://docs.opengeosys.org/docs).